driver-core/src/test/functional/com/mongodb/operation/MixedBulkWriteOperationSpecification.groovy

```diff
@@ -540,22 +540,28 @@ class MixedBulkWriteOperationSpecification extends OperationFunctionalSpecificat
         [async, ordered] << [[true, false], [true, false]].combinations()
     }
 
+    @Category(Slow)
     def 'should split the number of writes is larger than the match write batch size'() {
```

Comment:
This doesn't test all the limits in BsonWriterHelper#exceedsLimits

Reply:
Is this the only remaining open comment thread?

Reply:
Added some unit tests to CommandMessageSpecification. 

I believe this is the last open thread.